shell/config.gni

```diff
@@ -10,6 +10,7 @@ declare_args() {
   shell_enable_vulkan = false
 
   shell_enable_software = true
+  enable_vulkan_validation_layers = false
```

Comment:
It looks like there's a convention here to prefix the argument names with `shell_`.

Reply:
Moved it back to vulkan/config.gni